Cargo.toml

```diff
@@ -5,16 +5,16 @@ authors = ["Alex Davidson <coela@alxdavids.xyz>"]
 edition = "2018"
 
 [dependencies]
-base64 = "0.13"
 serde = { version="1.0.136", features = ["derive"] }
 serde_json = "1.0.59"
-ppoprf = { git = "https://github.com/brave-experiments/sta-rs", branch = "public-api-changes" }
-sta-rs = { git = "https://github.com/brave-experiments/sta-rs", branch = "public-api-changes" }
+ppoprf = { git = "https://github.com/brave-experiments/sta-rs" }
```

Comment:
@alxdavids alright if we use the `main` branch from now on?

Reply:
Ah, yes I think that should be correct. I expect that branch is very out-of-date now anyway.